runtime/tests/port/si.c

```diff
@@ -1855,17 +1858,40 @@ j9sysinfo_test_get_cwd3(struct J9PortLibrary *portLibrary)
 			outputErrorMessage(PORTTEST_ERROR_ARGS, "error failed to change current directory rc: %d\n", rc);
 	}
 #else /* defined(WIN32) */
-#if defined(OSX)
-	const char *utf8 = "/private/tmp/j9sysinfo_test_get_cwd3/";
-#else /* defined(OSX) */
-	const char *utf8 = "/tmp/j9sysinfo_test_get_cwd3/";
-#endif /* defined(OSX) */
+	const char *tmp = "/tmp/";
+	const char *utf8 = "j9sysinfo_test_get_cwd3/";
 
 	reportTestEntry(portLibrary, testName);
 
 	orig_cwd = (char*)j9mem_allocate_memory( EsMaxPath, OMRMEM_CATEGORY_PORT_LIBRARY );
 	j9sysinfo_get_cwd(orig_cwd, EsMaxPath);
 
+	/* /tmp may be a symlink or similar, get it's actual path first */
+#if defined(J9ZOS390)
+	rc = atoe_chdir(tmp);
+#else /* defined(J9ZOS390) */
+	rc = chdir(tmp);
+#endif /* defined(J9ZOS390) */
+	if (0 != rc) {
+		outputErrorMessage(PORTTEST_ERROR_ARGS, "cd %s failed rc: %d\n", tmp, rc);
```

Comment:
The test cannot continue; suggest `goto cleanup;` (adding that label and appropriate actions).